cluster/saltbase/salt/kube-master-addons/alarm

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
I think we've been naming scripts `.sh` more consistently, FWIW, rather than naked executables.